index.js

```diff
@@ -27,6 +33,7 @@ module.exports = app => {
       context,
       replacers: config.replacers
     })
+    config['sort-direction'] = validateSortDirection(config['sort-direction'])

```

Comment:
It's probably better to solve this with a Joi schema?

Reply:
I'll see if I can get around to it today :+1:

Reply:
No need to hurry! If you don't get around to it we can simply create an issue to improve this later :)